docusaurus/docs/core-concepts/clients/tunnelers/04-macos.md

```diff
@@ -1,3 +1,48 @@
 # MacOS
 
-The MacOS tunneler is currently in preview. Find the app in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+## Installation
+
+The MacOS tunneler is in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+
+![image](/img/clients/mac-install1.png)
+
+You will have the page to download the most recent Mac Ziti Desktop Edge. 
+
+![image](/img/clients/mac-install2.png) 
+
+:::info Note
+
+If you already have MAC Ziti Desktop already installed and you are updating it, please make sure to turn off your MAC Ziti Desktop Edge before updating it
+
+:::
+ 
+
+And after the download and installation is complete, click on **open**
+
+![image](/img/clients/mac-install3.png) 
+
+ 
+
+## Enrolling 
+
+Following this [guide](/docs/core-concepts/identities/creating) to create an identity and transfer the jwt file to your MAC.
+
+From the MAC Desktop Edge display, Click on **Turn Ziti On**. The user will need to click add an identity by clicking the **+** to choose the key downloaded a while ago when we started creating the endpoint.

```

Comment:
Glad to see this because I always have to tell people to turn it on.

Reply:
thanks.